drivers/clock_control/clock_stm32_ll_common.c

```diff
@@ -19,22 +18,22 @@
 #include "clock_stm32_ll_common.h"
 
 /* Macros to fill up prescaler values */
-#define z_ahb_prescaler(v) LL_RCC_SYSCLK_DIV_ ## v
-#define ahb_prescaler(v) z_ahb_prescaler(v)
+#define __ahb_prescaler(v) LL_RCC_SYSCLK_DIV_ ## v
```

Comment:
Txs for taking cafe of that, but `__` is even more reserved :-)
what about `fn` ?

Reply:
Damn, this was a bit too quick. Now using fn_